tests/lib/agent/Agent.test.ts

```diff
@@ -2,7 +2,7 @@ import os from 'os';
 import fs from 'fs';
 import path from 'path';
 import { randomString } from '../../../src/lib/utils';
-import Polykey, { PolykeyAgent, PolykeyClient } from '../../../src/lib/Polykey';
+import Polykey, { PolykeyAgent, PolykeyClient, KeyManager } from '../../../src/lib/Polykey';
```

Comment:
We can definitely use `@js-polykey` in our tests. As shown by typescript-demo-lib. Using aliases in our tests is not a problem!